app/Http/Controllers/InviteKeyController.php

```diff
@@ -13,13 +13,13 @@ class InviteKeyController extends Controller
     public function get(InviteKey $key)
     {
         if ($key->user()->count() > 0) {
-            response()->json([
-                'value' => 'De code \'' . $key->value . '\' is al in gebruik.'
-            ], 422)->throwResponse();
+            response()->json(['errors' => [
```

Comment:
Op deze manier stuurt een 422 error ook zijn errors terug, daarom heb ik dat hier op dezelfde manier geïmplementeerd. Het is anders erg lastig om met app verschillende soorten foutmeldingen te krijgen uit de response. Daarom hou ik het hier hetzelfde als de 422.